oasislmf/utils/defaults.py

```diff
@@ -59,6 +64,47 @@
     if cov_type in ['buildings', 'other', 'contents', 'bi']
 })
 
+SOURCE_FILENAMES = OrderedDict({
+    'loc': 'source_location.csv',
+    'acc': 'source_account.csv',
+    'info': 'source_info.csv',
+    'scope': 'source_scope.csv'
+})
+
+# Store index from merged source files (for later slice & dice)
+SOURCE_IDX = OrderedDict({
```

Comment:
I'm not clear on this dict - are these values, e.g.  `loc_idx` etc. column names in some file?

Reply:
column names for storing index of source files,